stestr/repository/util.py

```diff
@@ -12,15 +12,10 @@
 
 import importlib
 import os
-import sys
-import warnings
 
 
 def _get_default_repo_url(repo_type):
-    if repo_type == 'sql':
-        repo_file = os.path.join(os.getcwd(), '.stestr.sqlite')
-        repo_url = 'sqlite:///' + repo_file
-    elif repo_type == 'file':
+    if repo_type == 'file':
```

Comment:
I'm sorry for missing this at the first review. But I think we probably can remove this `_get_default_repo_url()` method and `repo_type` parameters in `get_repo_open()` and `get_repo_initialise()` because this method only accepts `repo_type=='file'` now, and we only use `repo_type=memory` internally.

Do we still need these `repo_type` for backward compatibility or something?

Reply:
I think you're right `_get_default_repo_url()` is definitely not needed anymore and the `repo_type` argument on the other functions isn't really used anymore.

I didn't remove the `repo_type` argument from `get_repo_open()` or `get_repo_initialise()` primarily because I was concerned about backwards compatibility. I don't remember if we explicitly made these functions part of the public python api or not, but since we had the deprecation warning for `repo_type='sql'` I figured it was better to be safe and not change the function signature in this PR.

Reply:
I think maybe before we release 4.0.0 we should deprecate the repo type argument. If you have time and want to open a PR for it that would be great.

Reply:
Thanks, yeah, I'll try that.